src/book/02-system/01-foundations/01-overview.mdx

```diff
@@ -64,16 +69,14 @@ The following is a table of roles that exist in the NUbots codebase and possible
 | `webots`              | Test the connection between the robot code and Webots.                                                                                                                                                 |
 | `webots_robocup`      | Play RoboCup games in the Webots simulator.                                                                                                                                                            |
 
-In a role, you should have only one platform. Possible platforms are `platform::darwin::HardwareIO`, `platform::darwin::HardwareSimulator`, `platform::Webots` and `platform::Gazebo`.
+In a role, you should have only one platform. Possible platforms are `platform::darwin::HardwareIO`, `platform::darwin::HardwareSimulator` and `platform::Webots`.
```

Comment:
Isn't the platform `cm740` ?

Reply:
Yeah I think it should be `platform::cm740::HardwareIO`, will check and update the page